packages/client/lib/commands/CONFIG_SET.ts

```diff
@@ -1,5 +1,15 @@
-export function transformArguments(parameter: string, value: string): Array<string> {
-    return ['CONFIG', 'SET', parameter, value];
+export function transformArguments(
+    parameter: string,
+    value: string,
+    mulitipleParams?: Array<[parameter: string, value: string]>
+): Array<string> {
```

Comment:
@leibale Is there a better way to implement it without breaking the code? :sweat_smile:

Reply:
@Avital-Fine checkout my commit... :)